### ISSUE TITLE
types are missing testcafe target as first arg

### ISSUE THREAD

**Issue by csi-lk** [percy/percy-testcafe#108]
[The types](https://github.com/percy/percy-testcafe/blob/master/types/index.d.ts) are:

```
export default function percySnapshot(  name: string,  options?: SnapshotOptions): Promise<void>;
```

But [the readme](https://github.com/percy/percy-testcafe/blob/master/README.md) states:

```
percySnapshot(t, 'TestCafe Example');
```

Looks like the first arg is missing

**Comment by jorgjanke** [comment]
yes, in the package, the file
testcafe/types/inded.d.ts
should look like 
```
export default function percySnapshot(
    t: TestController,
    name: string,
    options?: SnapshotOptions
): Promise<void>;
```

**Comment by csi-lk** [comment]
I've opened a PR to fix this here: #124
